Makefile.toml

```
@@ -98,6 +106,7 @@ args = [
 
 
 [tasks.test-windows]
+description = "Run tests on Windows"
```

Should this have `documentation` in the description?

```suggestion
description = "Run documentation tests on Windows"
```